src/System.Memory/tests/ParsersAndFormatters/Parser/ParserTests.cs

```diff
@@ -147,6 +147,9 @@ public static void FakeTestParserIntegerN(Type integerType)
             //
             try
             {
+                if (integerType == typeof(int))
```

Comment:
Should it run the test and then don't run the assert when type != int?

Reply:
I don't see any point - the entire test will be removed on the next PR (the one that implements 'N' for all 8 integer types.) The real testing is done by adding the 'N' cases to the theory data.

Reply:
ok